server/checkout/index.js

```diff
@@ -0,0 +1,226 @@
+/* eslint-disable camelcase */
```

Comment:
I don't know anything about what's going on here, but it does seem to work on my end when running with the stripe CLI listener. So, great!